README.md

```diff
@@ -3,6 +3,19 @@ ReadMe - Ubuntu Messaging App
 Ubuntu Messaging App is the official SMS app for Ubuntu Touch. We follow an open
 source model where the code is available to anyone to branch and hack on.
 
+Building with clickable
+=======================
+Install [clickable](http://clickable.bhdouglass.com/en/latest/), then app then can be build and deployed by simply running:
+
+```
+clickable
+```
+
+For faster build speeds, building app tests is disabled in ```clickable.json``` 
+
+_NOTE:_ As for now, building with clickable fails, due to libnotify dependencies missing in clickable/ubuntu-sdk:16.04-armhf docker image.
```

Comment:
If this issue is resolved, please update the README

Reply:
Updated in https://github.com/ubports/messaging-app/pull/73/commits/63ff1c433fb016f853a0901db68589da0bfd672e commit